src/code/fmodf.c

```diff
@@ -1,12 +1,27 @@
 #include "global.h"
 
+/**
+ * @brief Computes one @p x modulo @p y for floats.
```

Comment:
Looks like this one was missed in the changes?
```suggestion
 * Computes one `x` modulo `y` for floats.
```